src/com/dojo/foundations/parking/TheGarage.java

```diff
@@ -218,54 +166,74 @@ public static void main(String[] args) throws Exception {
 
                     break;
                 case 7:
-
+                    System.out.println("There are " + parkingLot.getParkingLotOccupation() + " cells occupied.");
                     break;
                 case 8:
-
+                    System.out.println("There are " + parkingLot.getFreeCells() + " free cells.");
+                    break;
+                default:
                     break;
             }
 
         }
 
     }
 
+    private static ParkingLot buildParkingLot() {
+        return new ParkingLot(Arrays.asList(
+            new Cell("M1", CellTypeEnum.MOTORCYCLE, true, Optional.empty()),
+            new Cell("M2", CellTypeEnum.MOTORCYCLE, true, Optional.empty()),
+            new Cell("M3", CellTypeEnum.MOTORCYCLE, true, Optional.empty()),
+            new Cell("M4", CellTypeEnum.MOTORCYCLE, true, Optional.empty()),
+            new Cell("M5", CellTypeEnum.MOTORCYCLE, true, Optional.empty()),
+            new Cell("M6", CellTypeEnum.MOTORCYCLE, true, Optional.empty()),
+            new Cell("M7", CellTypeEnum.MOTORCYCLE, true, Optional.empty()),
+            new Cell("M8", CellTypeEnum.MOTORCYCLE, true, Optional.empty()),
+            new Cell("M9", CellTypeEnum.MOTORCYCLE, true, Optional.empty()),
+            new Cell("M10", CellTypeEnum.MOTORCYCLE, true, Optional.empty()),
+            new Cell("M11", CellTypeEnum.MOTORCYCLE, true, Optional.empty()),
+            new Cell("M12", CellTypeEnum.MOTORCYCLE, true, Optional.empty()),
+            new Cell("C1", CellTypeEnum.CAR, true, Optional.empty()),
+            new Cell("C2", CellTypeEnum.CAR, true, Optional.empty()),
+            new Cell("C3", CellTypeEnum.CAR, true, Optional.empty()),
+            new Cell("C4", CellTypeEnum.CAR, true, Optional.empty()),
+            new Cell("C5", CellTypeEnum.CAR, true, Optional.empty()),
+            new Cell("C6", CellTypeEnum.CAR, true, Optional.empty()),
+            new Cell("C7", CellTypeEnum.CAR, true, Optional.empty()),
+            new Cell("C8", CellTypeEnum.CAR, true, Optional.empty()),
+            new Cell("T1", CellTypeEnum.TRUCK, true, Optional.empty()),
+            new Cell("T2", CellTypeEnum.TRUCK, true, Optional.empty()),
+            new Cell("T3", CellTypeEnum.TRUCK, true, Optional.empty()),
+            new Cell("T4", CellTypeEnum.TRUCK, true, Optional.empty())
+        ));
+    }
+    
+    private static void printGarageMenu() {
+        System.out.println("\"The Garage\"");
+        System.out.println("1) Motorcycle enters");
+        System.out.println("2) Car enters");
+        System.out.println("3) Truck enters");
+        System.out.println("4) Motorcycle leaves ");
+        System.out.println("5) Car leaves");
+        System.out.println("6) Truck leaves");
+        System.out.println("7) Get Parking lot occupation");
+        System.out.println("8) Get free cells");
```

Comment:
The options 7 are 8 are simple and very good.  I would like to enrich the functionality though.  What if you add some logic so that this returns the free cells for each vehicle below the total free cells.  The same for the busy cells in the option 7.